IdentityCore/src/logger/MSIDLogger.h

```diff
@@ -36,6 +36,20 @@ typedef NS_ENUM(NSInteger, MSIDLogLevel)
     MSIDLogLevelLast = MSIDLogLevelVerbose,
 };
 
+/*! Levels of log masking */
+typedef NS_ENUM(NSInteger, MSIDLogMaskingLevel)
+{
+    /** MSAL will not return any messages with any user or organizational information. This includes EUII and EUPI. */
```

Comment:
nit: we use common core in other libs too, not only MSAL.

Reply:
Addressed.